### PR TITLE
Auth credential termsOfUse

### PR DESCRIPTION
**Pull request by SamHellawell**

### CODE REVIEW

File: src/core/NavigationRouter.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ import {ImportWalletContainer} from '../features/wallet/ImportWalletScreen';
 import {ProtectYourWalletContainer} from '../features/wallet/ProtectYourWalletScreen';
 import {SetupPasscodeScreen} from '../features/wallet/SetupPasscodeScreen';
 import {DevSettingsContainer} from '../features/dev-settings/DevSettingsScreen';
-import {BuyDockScreenContainer} from '../features/trade/BuyDockScreen';
+// import {BuyDockScreenContainer} from '../features/trade/BuyDockScreen';
```
Comment (maycon-mello) [inline src/core/NavigationRouter.js:31]: Can we remove this commented line?

**Review by maycon-mello — CHANGES_REQUESTED** [review]
left some comments

File: src/core/NavigationRouter.js (hunk 1 above)
Comment (mike-parkhill) [reply src/core/NavigationRouter.js:31]: Does this remove the Transak integration?

**Review by mike-parkhill — COMMENTED** [review]
unit tests for these changes?

**Review by maycon-mello — APPROVED** [review]

**Pull request merged**